green/src/main/java/com/blockstream/green/views/GreenPinView.kt

```diff
@@ -44,6 +44,35 @@ class GreenPinView @JvmOverloads constructor(
             deletePinDigit(true)
             true
         }
+
+        binding.shuffleListener = OnClickListener { view ->
+            var digitsValues: Array<String> = arrayOf("1", "2", "3", "4", "5", "6", "7", "8", "9", "0")
+            var parentRoot = view.rootView
+            // get all PIN buttons
+            var button0 = parentRoot.findViewById<Button>(R.id.button0);
```

Comment:
nit: could you use an array instead 10 button variables? If so, you could assign digitaValues in a single loop